app/assets/stylesheets/sidebar.scss

```diff
@@ -0,0 +1,4 @@
+.pop-up {
+  position: absolute;
+  background-color: #ddd;
```

Comment:
Color literals like `#ddd` should only be used in variable declarations; they should be referred to via variable everywhere else.